community_images/consul/bitnami/scripts/server_coverage_script.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -x
+set -e
+
+# Available Scripts
+ls /opt/bitnami/scripts
+
+# consul acl token list
+# consul acl boostrap
+consul members
```

Comment:
we should also add snapshot command in the coverage

Reply:
Noted. I'm adding it right now.